docs/web3j-openapi.md

```diff
@@ -0,0 +1,248 @@
+# Web3j-OpenAPI
+
+[Web3j-OpenAPI](https://github.com/web3j/web3j-openapi) is an [OpenAPI](https://swagger.io/specification/) generator from [Solidity](https://solidity.readthedocs.io/) smart contracts. It provides a way to interact with the [Ethereum](https://ethereum.org/) blockchain via simple and intuitive HTTP requests. These interactions can be done using plain HTTP requests or via the [Swagger-UI](https://swagger.io/tools/swagger-ui/), which is generated with every project.
+
```

Comment:
You should start with a Quickstart section that basically uses the Epirus installer then creates a project.